### DISCUSSION TITLE
[WASM] PolyLine rendering fails when Points collection is changed dynamically (works when Points collection is static)

### DISCUSSION THREAD

**Discussion by ToddGlodek** [unoplatform/uno#10638] [General]
The goal is to build a charting control where the collection of vertices can be updated dynamically in a ViewModel.  The chart reacts to the ViewModel change - and rebuilds the Points collection of a PolyLine by inverting the Y-axis of each of the vertices so that the origin is in the lower-left (rather than upper-left).   Rescalling the size of the window - must result in the control being redrawn adaptively/fluidly to maintain the same Y-scale proportions - while allowing the X-scale to expand & contract.

This can be done using WinUI - but was WASM does not render the PolyLine when Points collection is changed dynamically.  Here a PolyLine with dynamically modified Points collection (Pink) is on WinUI - but not WASM.   Observe how the static line where the points collection is never changes (green) renders on both WinUI/WASM.

<img width="724" alt="PolylineIssue" src="https://user-images.githubusercontent.com/19764509/205451401-98e735a9-6b41-4002-a732-0213d805ee62.png">

This [repository contains the sample code](https://github.com/ToddGlodek/PolyLinePanelWasmTest).

**Comment by jeromelaban** [comment]
Thanks for the report, let's track in https://github.com/unoplatform/uno/issues/10651
